### ISSUE TITLE
Create PetOwner model

### ISSUE THREAD

**Issue by iBYarchitect** [ios-course/ironfoudation-team-project#20]
DoD:
id: UUID
name: String
surname: String
phone: String
avatar(image url): Url?
address: Adress
Rating: Double
Pets: [PetID]

**Comment by NadzeyaShpakouskaya** [comment]
Yes, we can make it optional, but we fill this info in apps in general. 
 Users can enter some random info, if we decide to make this field mandatory.
